packages/components/pie-chip/src/index.ts

```diff
@@ -11,10 +15,82 @@ const componentSelector = 'pie-chip';
 
 /**
  * @tagname pie-chip
+ * @slot icon - The icon slot
+ * @slot - Default slot
  */
 export class PieChip extends LitElement implements ChipProps {
+    @property()
+    @validPropertyValues(componentSelector, variants, 'primary')
+    public variant: ChipProps['variant'] = 'default';
+
+    @property({ type: Boolean })
```

Comment:
in other components, we have reflected the disabled prop. I don't know if it's needed as the state cannot be changed internally in this case but it may be worth considering (if you haven't already)

Reply:
that's an interesting question. so the chip element isn't a native button element, instead, a div with the role of `button`. referencing [mdn](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled),  div elements don't expect the disabled attribute - even if it were to be given a role="button".

on a side note, we still need to specify the `aria-disabled` attribute  for the disabled state but aria labels will be addressed in another ticket.

Reply:
Sounds good - definitely sounds like a component we will want to properly test on screenreaders!